src/main/drivers/pwm_output.c

```diff
@@ -220,6 +222,8 @@ void motorDevInit(const motorDevConfig_t *motorConfig, uint16_t idlePulse, uint8
     bool useUnsyncedPwm = motorConfig->useUnsyncedPwm;
     bool isDigital = false;
 
+    loadDmaBufferPtr = loadDmaBufferDshot;
```

Comment:
Make loading of address explicit (it's only syntactic sugar, but easier to read)
```
loadDmaBufferPtr = &loadDmaBufferDshot
```